govc/host/esxcli/executor.go

```diff
@@ -27,6 +26,23 @@ import (
 	"github.com/vmware/govmomi/vim25/xml"
 )
 
+type Fault struct {
+	Message string
+	Detail  string
+}
+
+func (f *Fault) Error() string {
```

Comment:
The receiver does not need to be by address, right? It's more efficient to treat this by-value.